audio/adlib_ms.h

```diff
@@ -1225,9 +1225,9 @@ class MidiDriver_ADLIB_Multisource : public MidiDriver_Multisource {
 	bool _rhythmMode;
 
 	// Pointer to the melodic instrument definitions.
-	OplInstrumentDefinition *_instrumentBank;
+	const OplInstrumentDefinition *_instrumentBank;
 	// Pointer to the rhythm instrument definitions.
-	OplInstrumentDefinition *_rhythmBank;
+	const OplInstrumentDefinition *_rhythmBank;
```

Comment:
Why would we want to make these two const?
They are accessed and updated by engine code, and making them const makes the end code ugly with no real gain

Reply:
As I understand it, the purpose of this PR is to make every statically define OplInstrumentDefinition objects constant (to have them located in .rodata instead of having them copied in .data at load time).
But this means that these pointers must be declared as pointing to const data.
This prevents some engines needing to it to update the tables.